modules/ROOT/pages/managing-instances/backup-restore-export.adoc

```diff
@@ -1,72 +1,157 @@
+
 == Snapshot types
 
 === Scheduled
 
 label:AuraDB-Professional[]
-label:AuraDB-Enterprise[]
+label:AuraDB-Business-Critical[]
+label:AuraDB-Virtual-Dedicated-Cloud[]
+label:AuraDS-Professional[]
+label:AuraDS-Enterprise[]
 
-A *Scheduled* snapshot is a snapshot that is automatically triggered when you first create an instance, when changes to the underlying system occur (for example, a new patch release), and at a cadence depending on your plan type.
+A *Scheduled* snapshot is a snapshot that is automatically triggered at a cadence depending on your plan type.
 
-Scheduled snapshots are run automatically once a day for Professional instances and once an hour for Business Critical and Enterprise instances.
-
-[NOTE]
-====
-For AuraDB Enterprise database instances running Neo4j v4.x, from day 0 to 7 scheduled snapshots run automatically once every 6 hours.
-From day 8 to 60, snapshots run once a day.
-====
+For Professional instances, scheduled snapshots are run automatically once a day, same for AuraDS instances, and for Business Critical and Virtual Dedicated Cloud instances, they run once an hour.
+See <<snapshot-details>> further on for information about frequency, restorability, and exportability of scheduled snapshots.
 
 === On Demand
 
-An *On Demand* snapshot is a snapshot that you manually trigger by selecting *Take snapshot* from the *Snapshots* tab of an instance.
+An *On Demand* snapshot is a snapshot that you manually trigger with the *Take snapshot* button.
 
 == Snapshot actions
 
-=== Restore
-
-[CAUTION]
-====
-Restoring a snapshot overwrites the data in your instance, replacing it with the data contained in the snapshot.
-====
-
-You can restore data in your instance to a previous snapshot by selecting *Restore* next to the snapshot you want to restore.
-
-Restoring a snapshot requires you to confirm the action by typing RESTORE and selecting *Restore*.
+image::snapshot-actions.png[width=400]
 
+[[export-create]]
 === Export/Create
 
-By selecting the ellipses (*...*) button next to an existing snapshot, you can:
+From the more (*...*) menu next to an existing snapshot, you can:
 
-* *Export* - Download the instance as *_.dump_* file, allowing you to store a local copy and work on your data offline.
+* *Export* - Download the instance as *_.backup_* file, allowing you to store a local copy and work on your data offline.
 * *Create instance from snapshot* - Create a new AuraDB instance using the data from the snapshot.
 
 [NOTE]
 ====
-The ability to Export or Create an instance from a Scheduled Enterprise snapshot is limited to 14 days.
+The ability to Export or Create an instance from a Scheduled Virtual Dedicated Cloud snapshot is limited to 14 days.
+
+Additionally, for Virtual Dedicated Cloud instances running Neo4j latest version, the ability to export or create an instance from a Scheduled snapshot is limited to the first full snapshot, taken once per day.
```

Comment:
what do we mean by "first full snapshot" here?

Reply:
is it the first snapshot taken each day, which is always full while the rest are diffs?

Reply:
This content was lifted from the previous docs, so I'm hoping you could tell me?

Reply:
Gotcha. So we run 23 diff backups and 1 full backup for BC and VDC, so we are going to have 1 exportable/restorable backup per day. The full backup is not necessarily the first backup to be taken on a given day, so we can omit `the first full snapshot` and replace it with something like `is limited to one snapshot per day` (as the remaining 23 are diffs).

Reply:
But again, this applies to 5.x/CalVer only. For 4.x this looks like this: 

VDC:
we take snapshots every 6 hours BUT after one week passes, we prune all except one backup per day.

E.g. we take 4 backups and exports (one every 6hrs = 4 per day)  today on Friday, all of these 4 backups and exports will be restorable for a week, but next Friday we will have pruned three backups and three exports out of these, and there will only be 1 backup left from today, Friday, which will be available for 60 days (restorable) and one export left, which will be available (exportable) for 14 days. 

Corresponding numbers from the table:
![Screenshot from 2025-02-14 15-21-42](https://github.com/user-attachments/assets/7578f9f1-bce4-4d10-9868-e390bf095b39)